pkg/reconciler/stats_reporter.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"context"
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+type Measurement int
+
+const (
+	// ServiceTimeUntilReadyM is the time it takes for a service to become ready since the resource is created
```

Comment:
We used to use this pattern if there are multiple measurements with the same signature/tags. However; for a single measurement, or for multiple measurements that have different signatures, it would be better to just skip this model and use a simpler thing (see https://github.com/knative/pkg/blob/master/controller/stats_reporter.go).

Reply:
Done